### DISCUSSION TITLE
Migrate from WordPress

### DISCUSSION THREAD

**Discussion by Vdriod** [danpros/htmly#806] [General]
This is not an issue but a question,  this is a great piece of work and I will love to use it but I don't see any tutorial on how to migrate from WordPress. Please tell me how

I am sorry if this question is already answered but I can't find it anywhere.
Thanks

**Comment by danpros** [comment]
Hi,

At the moment we can only use import RSS, a huge problems if we have lot of post for sure.

**Comment by Vdriod** [comment]
I use import RSS and only got 10 post from my WordPress front page, what about the remaining ones? I have over 500 post.

Is there anyway to import everything?

**Comment by AutomationMan808** [comment]
I'm no expert but in the UI configuration you you can change the number of posts in the RSS feed section
